pkg/modules/testdata/cr_auth_proxy_vault_cert.yaml

```diff
@@ -0,0 +1,117 @@
+          # for self-signed certs, leave empty string
+          # Allowed values: string
+          privateKey: ""
+
+          # proxy-server ingress will use this hostname
+          # NOTE: an additional hostname can be configured in proxyServerIngress.hosts
+          # NOTE: proxy-server ingress is configured to accept IP address connections so hostnames are not required
+          hostname: "csm-authorization.com"
+
+          # proxy-server ingress configuration
+          proxyServerIngress:
+            - ingressClassName: nginx
+
+              # additional host rules for the proxy-server ingress
+              hosts:
+                - authorization-ingress-nginx-controller.authorization.svc.cluster.local
+
+              # additional annotations for the proxy-server ingress
+              annotations: {}
+
+        - name: redis
+          redis: redis:6.0.8-alpine
+          commander: rediscommander/redis-commander:latest
+          redisName: redis-csm
+          redisCommander: redicommander
+          sentinel: sentinel
+          redisReplicas: 5
+          # by default, csm-authorization will deploy a local (https://kubernetes.io/docs/concepts/storage/storage-classes/#local) volume for redis
+          # to use a different storage class for redis, specify the name of the storage class
+          # NOTE: the storage class must NOT be a storage class provisioned by a CSI driver using this installation of CSM Authorization
+          # Default value: None
+          storageclass: "local-storage"
+
+        - name: vault
+          vaultAddress: https://10.0.0.1:8400
+          vaultRole: csm-authorization
+          skipCertificateValidation: true
+          kvEnginePath: secret
+          # certificate: base64-encoded certificate for cert/private-key pair -- add cert here to use custom certificates
+          #  for self-signed certs, leave empty string
+          # Allowed values: string
+          certificate: "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCmR1bW15Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0="
+          # privateKey: base64-encoded private key for cert/private-key pair -- add private key here to use custom certificates
+          #  for self-signed certs, leave empty string
+          # Allowed values: string
+          privateKey: "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCmR1bW15Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0="
+          # certificateAuthority: base64-encoded certificate authority for validating vault server certificate -- add certificate authority here to use custom certificates
+          #  for self-signed certs, leave empty string
+          # Allowed values: string
+          certificateAuthority: "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCmR1bW15Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0="
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: csm-config-params
+  namespace: authorization
+data:
+  csm-config-params.yaml: |
+    CONCURRENT_POWERFLEX_REQUESTS: 10
+    LOG_LEVEL: debug
```

Comment:
Do we need to add capacity poll interval here? Same for other yamls

Reply:
It doesn't break any existing tests or tests that I added. @KerryKovacevic might add a yaml with it for her work.